### PR TITLE
Move ecs node tf module to ansible role

### PR DESCRIPTION
**Pull request by Polina-Gubina**
Move ecs node tf module to ansible role

### CODE REVIEW

File: roles/ecs_node/files/first_boot.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```
Comment (anton-sidelnikov) [inline roles/ecs_node/files/first_boot.sh]: delete this file

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: roles/ecs_node/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+- name: Build infra
```
Comment (outcatcher) [inline roles/ecs_node/tasks/main.yaml]: Should be `Create ECS` and `Destroy ECS` respectively.

File: roles/ecs_node/vars/main.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+image: "{{ ecs_image }}"
+count: "{{ nodes_count }}"
+network_id: "{{ main_network_id }}"
+subnet_id: "{{ main_subnet_id }}"
+subnet_cidr: "{{ main_subnet_cidr }}"
+flavor: "{{ ecs_flavor }}"
+key_pair: "{{ key_pair_name }}"
+volume_size: "{{ disk_volume }}"
+scenario_name: "{{ scenario }}"
```
Comment (outcatcher) [inline roles/ecs_node/vars/main.yaml]: Please prefix role-specific variables with `ecs_`

File: roles/ecs_node/defaults/main.yaml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+availability_zone: eu-de-03
+disk_volume: 5
```
Comment (outcatcher) [inline roles/ecs_node/defaults/main.yaml]: Same as in `vars`

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: roles/ecs_node/tasks/build.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  openstack.cloud.security_group_rule:
+    security_group: "{{ secur_group.secgroup.name }}"
+    protocol: tcp
+    port_range_min: "{{ item }}"
+    port_range_max: "{{ item }}"
+    remote_ip_prefix: 0.0.0.0/0
+  loop:
+    - 22
+    - 80
+    - 443
```
Comment (anton-sidelnikov) [inline roles/ecs_node/tasks/build.yaml]: make var

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: roles/ecs_node/tasks/build.yaml [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    security_group: "{{ secur_group.secgroup.name }}"
+    protocol: icmp
+    port_range_min: -1
+    port_range_max: -1
+    remote_ip_prefix: 0.0.0.0/0
+
+- name: Create a new instance
+  openstack.cloud.server:
+    state: present
+    name: "{{ scenario_name }}_instance{{ item }}"
```
Comment (anton-sidelnikov) [inline roles/ecs_node/tasks/build.yaml]: lets add az in name
"{{ scenario_name }}\_instance\_{{ item }}_{{ az }}"

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by outcatcher — APPROVED** [review]

**Pull request merged**